loadtest-list.py

```diff
@@ -16,8 +16,8 @@
 NUM_SAMPLE_SHOTS = random.randint(4, 7)
 NUM_SEARCH_HITS = 2
 
-WEIGHT_LIST_SHOTS = int(os.getenv('WEIGHT_LIST_SHOTS', 0))
-WEIGHT_SEARCH_SHOTS = int(os.getenv('WEIGHT_SEARCH_SHOTS', 0))
+WEIGHT_LIST_SHOTS = int(os.getenv('WEIGHT_LIST_SHOTS') or '0')
+WEIGHT_SEARCH_SHOTS = int(os.getenv('WEIGHT_SEARCH_SHOTS') or '0')
```

Comment:
This is _slightly_ verbose and repetitive, but I was a bit reluctant to wrap it in a utils.py function since it may obscure the logic a bit much.

But up to you if you'd prefer this:

```py
WEIGHT_SEARCH_SHOTS = int(os.getenv('WEIGHT_SEARCH_SHOTS') or '0')
```

versus:

```py
WEIGHT_SEARCH_SHOTS = utils.env('WEIGHT_SEARCH_SHOTS', 0) # where `0` is default/fallback value
```